Source/Core/Core/Core.vcxproj.filters

```diff
@@ -634,9 +634,6 @@
     <ClCompile Include="PowerPC\SignatureDB.cpp">
       <Filter>PowerPC</Filter>
     </ClCompile>
-    <ClCompile Include="PowerPC\Jit64Common\Jit64AsmCommon.cpp">
-      <Filter>PowerPC\Jit64Common</Filter>
-    </ClCompile>
```

Comment:
Not sure we want to put Jit64AsmCommon.cpp into Jit64 category; there's probably going to be more files which move from JitCommon to Jit64Common in the near future.

The other changes look fine.